pkgs/by-name/ae/aerc/package.nix

```diff
@@ -85,12 +82,19 @@ buildGoModule rec {
     patchShebangs $out/libexec/aerc/filters
   '';
 
-  meta = with lib; {
+  nativeInstallCheckInputs = [ versionCheckHook ];
+  versionCheckProgramArg = "--version";
+  doInstallCheck = true;
+
+  passthru.updateScript = nix-update-script { };
+
+  meta = {
     description = "Email client for your terminal";
     homepage = "https://aerc-mail.org/";
-    maintainers = [ ];
+    changelog = "https://git.sr.ht/~rjarry/aerc/tree/${version}/item/CHANGELOG.md";
+    maintainers = with lib.maintainers; [ defelo ];
```

Comment:
Could you please add me to maintainers as well? I use aerc daily.

Reply:
done :+1: